hack/build-images.sh

```diff
@@ -138,6 +138,9 @@ image openshift/origin-sti-builder           images/builder/docker/sti-builder
 image openshift/origin-f5-router             images/router/f5
 image openshift/node                         images/node
 
+# images used for tests
+image openshift/test-multicast               images/testing/multicast
```

Comment:
@stevekuznetsov but do you think the general idea (building an image in images/ just for use in this test case) is right?

What is the exact set of conditions we'd want to build under? "Only when doing testextended" doesn't seem right because at some point we want the image to actually get pushed to docker. "All builds except jenkins builds that aren't doing testextended"? Are there existing env vars we can check that will pick out the right conditions or will we need to add new ones somewhere?

Although also, this image build just installs a single package on top of origin-base, so it's not going to add *much* time to the build if we do it always...